### ISSUE TITLE
508-defect-2: Download letters page headings are not logically ordered

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#45636]
⚠️ [blocked by this sitewide ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/50215)

## Point of contact

Josh Kim

## Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### User Story
As a screen reader user, I want to navigate the hierarchy of the page content using heading levels to save time and frustration.

### Problem
Two issues exist:
- "Please sign in..." heading should be a `h2`, not a `h3` to avoid a skipped heading
- "Download your VA letters on your mobile device..." should be a `h3`, not a `h2`
  - This can remain a `h2`, only if its parent "Get your VA letters online" is deleted

## Reproduction steps

1. [Go to the download letters page](https://www.va.gov/records/download-va-letters/)
2. [Using headingsMap](https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en), confirm headings are out of order

### Acceptance criteria
- [x] "Please sign in..." is a `h2`
- [ ] "Download your VA letters..." is a `h3` OR
  - [ ] "Get your VA letters online" is deleted (only if unnecessary)

## References, articles, or WCAG support

1. [Headings - Web Accessibility Tutorials](https://www.w3.org/WAI/tutorials/page-structure/headings/)

## Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [ ] Focus
- [X] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by kjduensing** [comment]
@joshkimux @jacobworrell I just submitted a [sitewide-content support issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/50215) on Github for this ticket. The headers mentioned above are not within our applications - they're part of the `static-pages` application.

**Comment by kjduensing** [comment]
@joshkimux I think I remember you being out so no rush! I'm not sure what next steps to take on this. Check out [sitewide-content support issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/50215) for details, but it sounds like they're not responsible for the 2 items mentioned. CC @jacobworrell

**Comment by kjduensing** [comment]
@joshkimux I think I remember you being out so no rush! I'm not sure what next steps to take on this. Check out [sitewide-content support issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/50215) for details, but it sounds like they're not responsible for the 2 items mentioned. CC @jacobworrell

**Comment by joshkimux** [comment]
@kjduensing apologies on the delay to respond 🙇 just wrapped up the SC audit. I double checked this, and it looks like this is a [`h2` in storybooks](https://design.va.gov/storybook/?path=/docs/components-va-alert--sign-in-or-tool-prompt#stories), not a `h3` 😕 I left a reply for Randi in that thread. 

If we can't figure out next steps, I'd recommend us going to platform office hours for support ❤️

**Comment by jaredcunha** [comment]
> I double checked this, and it looks like this is a [h2 in storybooks](https://design.va.gov/storybook/?path=/docs/components-va-alert--sign-in-or-tool-prompt#stories), not a h3 😕 I left a reply for Randi in that thread.

The heading levels aren't actually hardcoded into the component. The heading can be anything.

```jsx
<va-alert
  close-btn-aria-label="Close notification"
  status="continue"
  visible
>
  <h2
    id="track-your-status-on-mobile"
    slot="headline"
  >
    Track your claim or appeal on your mobile device
  </h2>
  <div>
    <p className="vads-u-margin-top--0">
      You can use our new mobile app to check the status of your claims or appeals on your mobile device. Download the{' '}
      <strong>
        VA: Health and Benefits
      </strong>
      {' '}mobile app to get started.
    </p>
    <button className="va-button-primary">
      Sign in to VA.gov
    </button>
  </div>
</va-alert>
```

**Comment by jerekshoe** [comment]
Yeah to add to what @jaredcunha said, [this example](https://design.va.gov/storybook/?path=/docs/components-va-alert--sign-in-or-tool-prompt#heading-level) highlights the fact that you can use an `h4` or any other header with the `va-alert` component

**Comment by jacobworrell** [comment]
Someone on our team is going to investigate where the header level is set in the code and assess the feasibility of our team making these changes. Depending on what we find, we may recommend another team owning these changes or go ahead and make the changes ourselves.

**Comment by HeatherWidmont** [comment]
Jared & Skyler will collaborate on new design changes & close either this ticket or the other one.
